scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java

```diff
@@ -631,6 +643,19 @@ protected boolean createExecution(NodeSet nodeSet, Node node, InternalJob job, I
 
     }
 
+    private void waitForTaskStartedNotification() {
+        finalizeStartingLock.lock();
+        try {
+            try {
+                finalizeStartingCondition.await();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove this call to "await" or move it into a "while" loop. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2274)